article.py

```diff
@@ -46,7 +46,7 @@ def save_raw(self):
                       indent=4,
                       ensure_ascii=False,
                       separators=(',', ': '))
-    
+
```

Comment:
discard changes

Reply:
but without these changes, the style check will not pass...